pwa/package.json

```diff
@@ -6,12 +6,12 @@
   "scripts": {
     "build": "remix vite:build",
     "gcp-build": "",
-    "dev": "remix vite:dev",
+    "dev": "node ./server.js",
     "lint": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint .",
     "lint:fix": "eslint --ignore-path .gitignore --cache --cache-location ./node_modules/.cache/eslint . --fix",
     "format": "prettier --check .",
     "format:fix": "prettier --write .",
-    "start": "remix-serve ./build/server/index.js",
+    "start": "cross-env NODE_ENV=production node ./server.js",
```

Comment:
Now that we are calling node more directly, can we pin a specific node version in package.json engine field?

Reply:
20.16.0 seems to be the latest on 20